core/src/test/scala/unit/kafka/admin/ListConsumerGroupTest.scala

```diff
@@ -64,28 +64,89 @@ class ListConsumerGroupTest extends ConsumerGroupCommandTest {
     val service = getConsumerGroupService(cgcArgs)
 
     val expectedListing = Set(
-      new ConsumerGroupListing(simpleGroup, true, Optional.of(ConsumerGroupState.EMPTY)),
-      new ConsumerGroupListing(group, false, Optional.of(ConsumerGroupState.STABLE)))
+      new ConsumerGroupListing(simpleGroup, true)
```

Comment:
Do we need all the changes in this test? It may be better to keep it as it was.

Reply:
yeah the changes were made due to the constructor changes to ConsumerGroupListing, I guess we'll change it everywhere if we decide to change it there